### PR TITLE
Check for print selected event

### PR DESCRIPTION
**Pull request by hbrace**
Do not merge until [decrypting sftp files](https://github.com/ONSdigital/census-rm-acceptance-tests/pull/81) is merged in.

# Motivation and Context

We want to be able to see the print file selected events for a case.

# What has changed

Step added to check PRINT_CASE_SELECTED events matching a specified pack code are returned by retrieving the case details using the case-api service.

# How to test?

Build images from the following:
action-scheduler service - case-history-updates branch
case-processor service - update-case-history branch
case-api service - new-case-events branch
print-file-service - add-distribution-functionality branch

Run using rm-docker-dev - Removing-action-exporter branch

run these acceptance tests

# Links

https://trello.com/c/stICnQSn/922-rmc-212-case-processor-update-case-history-when-added-to-print-file-13#
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[RMC-212 Case Processor - Update case history when added to print file (13) **](https://trello.com/c/stICnQSn/922-rmc-212-case-processor-update-case-history-when-added-to-print-file-13)

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: acceptance_tests/features/sample_file_smoke.feature [hunk 1]
```diff
@@ -6,6 +6,7 @@ Feature: Checks that input sample files and action rules results in correct prin
     Then messages are emitted to RH and Action Scheduler with [01] qids
     And correctly formatted "P_IC_ICL1" print files are created
     And there is a correct "P_IC_ICL1" manifest file for each csv file written
+    And events of pack code "P_IC_ICL1" are logged against the case
```
Comment (liamtoozer) [inline acceptance_tests/features/sample_file_smoke.feature]: Can't remember what was decided but should we add this step in for each scenario to check each different pack code?

Comment (hbrace) [reply acceptance_tests/features/sample_file_smoke.feature]: Added steps!

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**